Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/ChangeFeed/FeedProcessorCoreTests.cs

```diff
@@ -58,6 +58,38 @@ public async Task UsesCustomSerializer()
             Assert.AreEqual(1, serializer.FromStreamCalled, "Should have called FromStream on the custom serializer");
         }
 
+        [TestMethod]
+        public async Task ReadAsyncExpectedTimeoutTest()
+        {
+            ChangesHandler<MyDocument> handler = (changes, cancelationToken) =>
+            {
+                IReadOnlyList<MyDocument> list = changes as IReadOnlyList<MyDocument>;
+                Assert.IsNotNull(list);
+                Assert.AreEqual("test", list[0].id);
+                return Task.CompletedTask;
+            };
+            Mock<PartitionCheckpointer> mockCheckpointer = new Mock<PartitionCheckpointer>();
+            Mock<FeedIterator> mockIterator = new Mock<FeedIterator>();
+            mockIterator.Setup(i => i.ReadNextAsync(It.IsAny<CancellationToken>()))
+                .Returns(async () =>
+                {
+                    await Task.Delay(2000);
```

Comment:
Let's not wait 2 seconds if the timeout is 100ms, the faster the tests the better :) If we can lower these values, awesome.

```suggestion
                    await Task.Delay(200);
```